### DISCUSSION TITLE
UpdateHub v2.1.0

### DISCUSSION THREAD

**Discussion by onyx-nxt** [NexovaDev/UpdateHub#94] [Announcements]
## Version
`2.1.0`

# Changelog
- **Fix `Timeout reached` error for non English users (https://github.com/KK-Designs/UpdateHub/issues/6) 🎉**
- Fixed an issue with the tray context menu overlapping the tray icon. (Fixes [6#issuecomment-1645236796](https://github.com/KK-Designs/UpdateHub/issues/6#issuecomment-1645236796)) 
- Fixed non standard version apps from not showing on the app update page
- Fixed the app getting stuck on the app update page on startup
- Improved error logs to make it easier to debug
- Changed the minimum winget version to be 1.4.11071
- App update page is now more stable
- And more bug fixes

# How to update
Click the Update Available button in the sidebar (if you don't see it force close the app by pressing the shift key and then click the X icon in the windows title bar and then relaunch the app)
<img src="https://user-images.githubusercontent.com/71038229/229697691-43ef66a0-a29d-4e06-af4b-4df0ab5eb772.png" width="572" />
And then wait for it to download the update file, and it will install automatically. You now have the latest version!

### VirusTotal scan
This app is not a virus, trust me. If you want, here's a [VirusTotal scan](https://www.virustotal.com/gui/file/09298911be1428b5c465063320b8485cdd41d426d54ff88b92f35485e2ad0018 "VirusTotal Scan"). As you can see, no anti-virus software has flagged it as a virus. The executable below's SHA-256 Hash is `09298911BE1428B5C465063320B8485CDD41D426D54FF88B92F35485E2AD0018`, which matches the VirusTotal scan.

### Remember, if you encounter any bugs in UpdateHub, report them so we can fix them!

# Thanks for downloading!

### SHA256
`09298911BE1428B5C465063320B8485CDD41D426D54FF88B92F35485E2AD0018`

<hr /><em>This discussion was created from the release <a href='https://github.com/KK-Designs/UpdateHub/releases/tag/v2.1.0'>UpdateHub v2.1.0</a>.</em>

**Comment by Deses** [comment]
The update doesn't work. After it downloads the 64Mb, it restarts back to 2.0.0 and it still asking for an update to 2.1.0.

I'll try uninstalling 2.0.0 and reinstalling 2.1.0.

**Comment by tute123456** [comment]
thx
